test/blackbox-tests/test-cases/dialects/run.t

```diff
@@ -1,21 +1,17 @@
 Test the (dialect ...) stanza inside the dune-project file.
 
-  $ dune build --root good --display short @install
+  $ dune exec --root good ./main.exe
+  Entering directory 'good'
+  Entering directory 'good'
+
+  $ dune build --root good --display short @fmt
```

Comment:
Should this `--display short` be removed as well? The output contains some `.o` extensions. Another way to check that `fmt.exe` is being executed is to make `fmt.exe` print something.